app/controllers/spree/api/v2/products_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Spree
+  module Api
+    module V2
+      class ProductsController < ::Spree::Api::V2::BaseController
```

Comment:
why do we override products controller in kiosk app?

Reply:
It is for returning the products added to the display.

Reply:
i dont get it.. we can fecth display products from display show api only with include paramaters.. instead of over ride in product controller